tests/test_server_tls.py

```
@@ -0,0 +1,94 @@
+    try:
+        import riffq  # noqa: F401
+        return
+    except ImportError:
+        pass
+
+    try:
+        subprocess.check_call(["maturin", "build", "--release", "-q"])
+        wheel = next(Path("target/wheels").glob("riffq-*.whl"))
+        subprocess.check_call([sys.executable, "-m", "pip", "install", str(wheel)])
+    except Exception as exc:
+        raise RuntimeError(f"riffq build failed: {exc}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)